saga/seata-saga-engine-store/src/main/java/io/seata/saga/engine/serializer/impl/ExceptionSerializer.java

```diff
@@ -38,24 +39,14 @@
 
         byte[] result = null;
         if (o != null) {
-            ObjectOutputStream oos = null;
-            try {
-                ByteArrayOutputStream baos = new ByteArrayOutputStream();
-                oos = new ObjectOutputStream(baos);
+            try (ByteArrayOutputStream baos = new ByteArrayOutputStream();
+                ObjectOutputStream oos = new ObjectOutputStream(baos)) {
```

Comment:
The close method of ByteArrayOutputStream actually does nothing.The close method of ObjectOutputStream should be called.